bin/run-dev

```diff
@@ -38,14 +38,12 @@ function set_args() {
 
 set_args "$@"
 
-echo "Making sure we're up to date with the latest dev... "\
-  "set environment variable USE_LOCAL_CIVIFORM=1 to skip"
-bin/pull-image > /dev/null
+bin/pull-image
 
 docker::set_network_name_dev
 
-# Start emulator.
-docker-compose --profile "${cloud_provider}" \
+# Start containers w/ specified emulator.
+docker compose --profile "${cloud_provider}" \
```

Comment:
The script fails for me because of the  removal of the - in docker-compose.

Reply:
yeah, it requires everyone be on a more recent version of docker https://docs.docker.com/compose/cli-command/#install-on-linux

Reply:
Can confirm. Installing docker-compose v2 as a plugin works great!